frontend/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -231,15 +253,20 @@ class DateInput extends React.PureComponent<Props, State> {
             CalendarContainer: {
               style: {
                 fontSize: fontSizes.sm,
+                paddingRight: theme.spacing.sm,
+                paddingLeft: theme.spacing.sm,
+                paddingBottom: theme.spacing.sm,
+                paddingTop: theme.spacing.sm,
               },
             },
             Week: {
               style: {
-                fontSize: fontSizes.sm,
+                fontSize: fontSizes.twoSm,
               },
             },
             Day: {
-              style: () => ({
+              style: ({ $selected }: { $selected: boolean }) => ({
```

Comment:
Two things here: I think you can remove `$selected` here since it is unused. And I would change fontsize to `sm` and add a line height. Basically, I think you can replace this with:

```typescript
              style: {
                fontSize: fontSizes.sm,
                lineHeight: lineHeights.base,
                "::after": {
                  borderColor: colors.transparent,
                },
              },
```

The `lineHeights` needs to be imported from the theme above:

`const { colors, fontSizes, lineHeights } = theme`

-> by this, the numbers in the selection circle will be centered.

Reply:
Thank you 👍 It's changed 👍